TLS_E2E_Automation_Framework/cypress/e2e/tls_page_elements/welcomeBanner.js

```diff
@@ -0,0 +1,105 @@
+require("cypress-xpath");
+import * as performLogin from "./loginLogout.js";
+
+// Selector constants
+const WELCOME_MESSAGE_BANNER = ".tls-home-page__hero-banner";
+const CLOSE_BUTTON_XPATH =
+  "//div[@class='tls-welcome-message-banner']/span[@role='button']";
+const ARTICLE_PAGE_WELCOME_BANNER = ".tls-welcome-message-banner";
+const ARTICLE_LINK = ".tls-card-headline";
+const TLS_LOGO = ".tls-header-navigation__logo";
+const APP_BUTTON = ".tls-welcome-message-banner__wrapper-image-container";
+const CLOSED_BANNER_XPATH =
+  "//div[@class='tls-home-page__hero-banner-wrapper']";
+
+// Environment-specific
+const environment = Cypress.env("ENV") || "prod";
+const url = Cypress.env(`${environment}_url`);
+
+/**
+ * Validates the welcome banner behavior for a guest user.
+ */
+export function welcomeBannerGuest() {
+  // Visit the homepage and accept the cookie banner
+  cy.visit(url, { timeout: 20000 });
+  cy.acceptCookieBanner();
```

Comment:
Instead calling every fucntions, better to add in before each

Reply:
updated